src/invidious/views/components/search_box.ecr

```diff
@@ -5,5 +5,6 @@
 		name="q" placeholder="<%= translate(locale, "search") %>"
 		title="<%= translate(locale, "search") %>"
 		value="<%= env.get?("search").try {|x| HTML.escape(x.as(String)) } %>">
+		<input type="submit" value="<%= translate(locale, "search") %>">
```

Comment:
```suggestion
		<button class="search-button" id="right-overlap-search-button" type="submit"><i class="icon ion-md-search"></i></button>
```

Reply:
I like the suggestion but an `aria-label` would need to be added to that button to make it accessible for visually impaired users

`... type="submit" aria-label="<%= translate(locale, "search") %>"><i ...`